client/my-sites/comments/controller.js

```diff
@@ -58,10 +58,7 @@ export const siteComments = context => {
 
 	const status = mapPendingStatusToUnapproved( params.status );
 
-	const pageNumber = sanitizeInt( query.page );
-	if ( ! pageNumber ) {
-		return changePage( path )( 1 );
-	}
+	const pageNumber = sanitizeInt( query.page ) || 1;
```

Comment:
Since we have a helper function, why don't we move this to `sanitizeInt`

Reply:
Because in other cases we rely on `sanitizeInt` to return `false`.